Rmdb.Domain.Services/Profiles/ActorProfile.cs

```diff
@@ -0,0 +1,20 @@
+﻿using AutoMapper;
+using Rmdb.Domain.Dtos.Actors;
+using Rmdb.Domain.Model;
+using System.Linq;
+
+namespace Rmdb.Domain.Services.Profiles
+{
+    public class ActorProfile : Profile
+    {
+        public ActorProfile()
+        {
+            CreateMap<Person, ActorListDto>()
+                .ForMember(x => x.FullName, opt => opt.MapFrom(y => $"{y.Name} {y.LastName}"));
```

Comment:
I don't like x, y lambda namings.... The next person wont know what you're talking about